examples/managed_node_groups/main.tf

```diff
@@ -0,0 +1,104 @@
+terraform {
+  required_version = ">= 0.12.0"
```

Comment:
This should match what's in the versions.tf file (which actually should have been bumped to `>= 0.12.6` when the module started using `for_each` in `workers.tf` and `workers_launch_template.tf`).

Reply:
done